dot/node.go

```diff
@@ -180,7 +180,7 @@ func NodeInitialized(basepath string) bool {
 }
 
 // LoadGlobalNodeName returns the stored global node name from database
-func LoadGlobalNodeName(basepath string) (nodename string, err error) {
+func LoadGlobalNodeName(basepath string) (string, error) {
```

Comment:
Please let's use named returned values, it's really useful for readers/documentation.

Reply:
updated.